installer/templates/new/mix.exs

```diff
@@ -34,9 +34,11 @@ defmodule <%= application_module %>.Mixfile do
   defp deps do
     [<%= phoenix_dep %>,<%= if ecto do %>
      {:phoenix_ecto, "~> 1.1"},
+     {:ecto, github: "elixir-lang/ecto", override: true},
```

Comment:
We should hold this PR until we get a new phoenix_ecto that depends on ecto 1.1.